Incompatibilty with blaze-html >= 0.9.2.0

Unfortunately `blaze-html` started importing a conflicting `template` function. (see [Changelog](https://hackage.haskell.org/package/blaze-html-0.9.2.0/changelog)).

```
[19 of 19] Compiling Eventlog.HtmlTemplate ( src/Eventlog/HtmlTemplate.hs, dist/build/Eventlog/HtmlTemplate.o, dist/build/Eventlog/HtmlTemplate.dyn_o )

src/Eventlog/HtmlTemplate.hs:214:16: error:
    Ambiguous occurrence ‘template’
    It could refer to
       either ‘H.template’,
              imported from ‘Text.Blaze.Html5’ at src/Eventlog/HtmlTemplate.hs:14:1-39
           or ‘Eventlog.HtmlTemplate.template’,
              defined at src/Eventlog/HtmlTemplate.hs:137:1
    |
214 |   renderHtml $ template x as $ allTabs x as
    |                ^^^^^^^^
```
[Source of build error](https://hydra.nixos.org/build/253351290/nixlog/1/tail)

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,7 @@
+Unreleased
+
+* Fix build error due to new exported function in blaze-html [#186](https://github.com/mpickering/eventlog2html/issues/186)
+
 0.11.0 release 2024-02-06
 
 * Add sorting by numbers [#183](https://github.com/mpickering/eventlog2html/pull/183)

--- a/src/Eventlog/HtmlTemplate.hs
+++ b/src/Eventlog/HtmlTemplate.hs
@@ -211,7 +211,7 @@ templateString :: EventlogType
                -> Args
                -> String
 templateString x as =
-  renderHtml $ template x as $ allTabs x as
+  renderHtml $ Eventlog.HtmlTemplate.template x as $ allTabs x as
 
 
 ppHeapProfileType :: HeapProfBreakdown -> Text

